### DISCUSSION TITLE
rag embedded model recommendation and local llm model for document translation.

### DISCUSSION THREAD

**Discussion by arjunkrishna** [open-webui/open-webui#1506] [General]
Hello,
   I was wondering which local llm model does the open-webui community uses to translate pdf documents to english. I hosted few models on ollama on a machine having rtx 4090 gpu. It is a hit or a miss with translation.

  Which rag embedding model do you use that can handle multi-lingual documents, I have not overridden this setting in open-webui, so I am using the default embedded model that open-webui uses.

Open WebUI Version
v0.1.117

Ollama Version
0.1.30

Thanks,
Arjun

**Comment by jannikstdl** [comment]
Look at this

https://huggingface.co/intfloat/multilingual-e5-large
